spec.html

```diff
@@ -126,81 +126,81 @@
-    <li>`Noncharacter_Code_Point`</li>
-    <li>`Other_Alphabetic`</li>
-    <li>`Other_Default_Ignorable_Code_Point`</li>
-    <li>`Other_Grapheme_Extend`</li>
-    <li>`Other_ID_Continue`</li>
-    <li>`Other_ID_Start`</li>
-    <li>`Other_Lowercase`</li>
-    <li>`Other_Math`</li>
-    <li>`Other_Uppercase`</li>
-    <li>`Pattern_Syntax`</li>
-    <li>`Pattern_White_Space`</li>
-    <li>`Prepended_Concatenation_Mark`</li>
-    <li>`Quotation_Mark`</li>
-    <li>`Radical`</li>
-    <li>`Sentence_Terminal`</li>
-    <li>`Soft_Dotted`</li>
-    <li>`Terminal_Punctuation`</li>
-    <li>`Unified_Ideograph`</li>
-    <li>`Uppercase`</li>
-    <li>`Variation_Selector`</li>
-    <li>`White_Space`</li>
-    <li>`XID_Continue`</li>
-    <li>`XID_Start`</li>
+    <li><a href="http://unicode.org/reports/tr44/#Alphabetic">`Alphabetic`</a></li>
+    <li><a href="http://unicode.org/reports/tr18/#General_Category_Property">`Any`</a></li>
+    <li><a href="http://unicode.org/reports/tr18/#General_Category_Property">`ASCII`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#ASCII_Hex_Digit">`ASCII_Hex_Digit`</a></li>
+    <li><a href="http://unicode.org/reports/tr18/#General_Category_Property">`Assigned`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#Bidi_Control">`Bidi_Control`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#Bidi_Mirrored">`Bidi_Mirrored`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#Case_Ignorable">`Case_Ignorable`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#Cased">`Cased`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#CWCF">`Changes_When_Casefolded`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#CWCM">`Changes_When_Casemapped`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#CWL">`Changes_When_Lowercased`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#CWKCF">`Changes_When_NFKC_Casefolded`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#CWT">`Changes_When_Titlecased`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#CWU">`Changes_When_Uppercased`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#Composition_Exclusion">`Composition_Exclusion`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#Dash">`Dash`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#Default_Ignorable_Code_Point">`Default_Ignorable_Code_Point`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#Deprecated">`Deprecated`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#Diacritic">`Diacritic`</a></li>
+    <li><a href="http://unicode.org/reports/tr51/#Emoji_Properties">`Emoji`</a></li>
+    <li><!-- Note: by the time this makes it into ECMA-262, the draft defining `Emoji_Component` will likely have been published already. Update the following link accordingly when that happens. --><a href="http://unicode.org/reports/tr51/proposed.html#Emoji_Properties">`Emoji_Component`</a></li>
```

Comment:
> How certain is this proposal to be accepted?

All I know is that `Emoji_Component` has been included in the past two draft snapshots.

> We could use this as a test case, leaving it out and adding it back in in round two. 

I’d be okay with that. @bterlson, what do you think?